client/landing/subscriptions/tab-views/sites.tsx

```diff
@@ -0,0 +1,58 @@
+import { Gridicon } from '@automattic/components';
+import { useTranslate } from 'i18n-calypso';
+import './sites.scss';
+
+// Mock meanwhile we have the data endpoints ready
+const useSites = () => {
+	const sites = [];
+	for ( let i = 0; i < 10; i++ ) {
+		sites.push( {
+			id: i,
+			name: `Site ${ i }`,
+			icon: `https://www.gravatar.com/avatar/${ i }?s=96&d=mm`,
+			url: `https://site${ i }.wordpress.com`,
+			date: new Date(),
+			newPostNotifications: [ 'Notifications', 'Email' ][ Math.floor( Math.random() * 2 ) ],
+			newCommentNotifications: Math.random() > 0.5,
+			emailFrequency: [ 'Daily', 'Weekly', 'Immediately' ][ Math.floor( Math.random() * 3 ) ],
+		} );
+	}
+	return sites;
+};
+
+export default function SitesView() {
+	const translate = useTranslate();
+	const sites = useSites();
+
+	const commentYes = <Gridicon icon="checkmark" size={ 18 } className="green" />;
```

Comment:
The `commentYes` and `commentNo` should be defined as a component outside the render of the `SitesView` component.

Reply:
Not required now.